### PR TITLE
finished get_reviews

### PR DESCRIPTION
**Pull request by angelikavilde**

### CODE REVIEW

File: live_pipeline/get_reviews.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+"""Retrieves reviews for a game from game IDs"""
+
+from urllib.parse import quote_plus
+from datetime import datetime
+
+import requests
+import pandas as pd
```
Comment (DMcCallion) [inline live_pipeline/get_reviews.py]: Nice seperation of imports, not sure how much it matters but they're not alphabetical

File: live_pipeline/get_reviews.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+
+def get_review_info_for_game(game_id: int) -> dict:
+    """Retrieves information about all reviews from a given game ID"""
```
Comment (gawsalya) [inline live_pipeline/get_reviews.py:11]: I think you might want to put in an exception here incase nothing is returned? Although yours should be functioning on a known game_id

File: live_pipeline/get_reviews.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    return next_cursor, page_reviews
+
+
+def get_all_reviews(game_ids: list[int]) -> None:
+    """Creates a csv file for all review info and
+    reviews for each game csv file"""
+    review_info = []
+
+    for game in game_ids:
+        review_info.append(get_review_info_for_game(game))
```
Comment (gawsalya) [inline live_pipeline/get_reviews.py]: if nothing is returned do we want it to append nothing or just do nothing?

**Review by DMcCallion — APPROVED** [review]

File: live_pipeline/get_reviews.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                break
+            if not cursor in cursor_list:
+                cursor_list.append(cursor)
+            all_reviews.extend(page_reviews)
+    pd.DataFrame(all_reviews).to_csv("reviews.csv")
+    pd.DataFrame(review_info).to_csv("review_info.csv")
+
+
+if __name__ == "__main__":
+    game_ids = get_game_ids() # TODO connect to script with list of game IDs
```
Comment (gawsalya) [inline live_pipeline/get_reviews.py]: this would connect to database -- gotten as a select statement

File: live_pipeline/get_reviews.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    for review in reviews["reviews"]:
+        review_dict = dict()
+        review_dict["game_id"] = game_id
+        review_dict["review"] = review["review"]
+        review_dict["review_score"] = review["votes_up"]
+        review_dict["last_timestamp"] = datetime.fromtimestamp(review["timestamp_updated"]).strftime("%Y-%m-%d %H:%M:%S")
+        review_dict["playtime_at_review"] = review["author"]["playtime_at_review"]
+        review_dict["full_playtime"] = review["author"]["playtime_forever"]
+        page_reviews.append(review_dict)
+    return next_cursor, page_reviews
```
Comment (rhKashif) [inline live_pipeline/get_reviews.py]: Return in a more explicit form: Dictionary

File: live_pipeline/get_reviews.py [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+        for page in range(int(review_info[-1]["number_of_total_reviews"]/100)+2):
+            cursor, page_reviews = get_reviews_for_game(game, cursor_list[page])
+            if not page_reviews or cursor in cursor_list:
+                break
+            if not cursor in cursor_list:
+                cursor_list.append(cursor)
+            all_reviews.extend(page_reviews)
+    pd.DataFrame(all_reviews).to_csv("reviews.csv")
+    pd.DataFrame(review_info).to_csv("review_info.csv")
```
Comment (rhKashif) [inline live_pipeline/get_reviews.py]: Separate this out into its own function

**Pull request merged**